### ISSUE TITLE
Basic compute_metrics function fails on Google Colab

### ISSUE THREAD

**Issue by seungguini** [Maluuba/nlg-eval#121]
Hey,

I've wanted to test the code on Google colab, but the code throws an **error**

Here's the code I'm trying to run:

```Python
from nlgeval import compute_metrics
metrics_dict = compute_metrics(hypothesis='nlg-eval/examples/hyp.txt', references=['nlg-eval/examples/ref1.txt', 'nlg-eval/examples/ref2.txt'])
```

Here's the Error log:
![image](https://user-images.githubusercontent.com/32915706/121769487-d1333f00-cb9e-11eb-81fe-175c42d9015d.png)

I've also made the Google Colab link publicly available - please check it out.

https://colab.research.google.com/drive/1iLiy8bQcR65ll6HufAvxzXExk7kn5Ls2?usp=sharing

I'd appreciate any help / fixes! Thanks.

**Comment by juharris** [comment]
Thanks for trying nlg-eval! I was able to reproduce this with your notebook, so thanks for sharing that. I didn't see anything immediately crazy wrong. I was looking to make sure that Theano and gensim did get installed and their versions. The Theano version was 1.0.5 but I tried `!pip install 'Theano~=0.8.1'` and I still got the same error.

**Comment by AbhilashPal** [comment]
Hi, so I was also facing the same error. For me, the pytest run is also throwing a few errors. 

Here's the code with the issue : 
https://colab.research.google.com/drive/1nGfXh3Ae4vOxjQW039RsjAqgOF_fjsrZ?usp=sharing

**Comment by seungguini** [comment]
@juharris `nlg-eval` seems to work fine on my local computer. Perhaps it's a compatibility issue with Google Colab?

**Comment by juharris** [comment]
Yeah I think it's using the wrong version of one of the dependencies somehow.
